casanovo/denovo/model_runner.py

```diff
@@ -107,6 +114,24 @@ def __init__(
             ),
         ]
 
+        # Configure early stopping
+        if config.early_stopping_patience is not None:
+            self.callbacks.append(
+                EarlyStopping(
```

Comment:
Removing for now. This will be introduced back into `casanovo/dev` by a future pr if it is decided to introduce early stopping functionality into the mainline casanovo release.